frontend/src/pages/profile/profile.js

```diff
@@ -34,6 +34,10 @@ export default {
                 this.errors.firstname = "Firstname required";
             if (!this.user.lastname)
                 this.errors.lastname = "Lastname required";
+            if (!this.user.email)
+                this.errors.email = "Email required";
+            if (!this.user.phone)
+                this.errors.phone = "Phone Number required";
```

Comment:
Same as the collaborator, you probably want to leave the platform as flexible as possible. Just my opinion, I am not the maintainer, simply trying to help!

Reply:
These are good to have fields and don't have any regex based validation so can be replaced with any text if not used

Reply:
I understand, but why having to put 555-555-5555 (or wtv string) as a place holder for a phone number when what you really want is to leave it blank? Your comment makes it sound to me like it is not mandatory, so lets have the UI reflect this reality?